src/web/mjs/connectors/templates/MangAdventure.mjs

```diff
@@ -0,0 +1,89 @@
+import Connector from '../../engine/Connector.mjs';
+import Manga from '../../engine/Manga.mjs';
+
+export default class MangAdventure extends Connector {
+    constructor() {
+        super();
+        super.id = undefined;
+        super.label = undefined;
+        this.url = undefined;
+        this.requestOptions.headers.set('x-user-agent', this.userAgent);
+    }
+
+    get platform() {
+        switch (process.platform) {
+            case 'win32':
+            case 'cygwin':
+                return 'Windows';
+            case 'linux':
+                return 'Linux';
+            case 'darwin':
+                return 'Macintosh';
+            default:
+                return process.platform;
+        }
+    }
+
+    get userAgent() {
+        return `Mozilla/5.0 (${this.platform}) Chrome/${process.versions.chrome} Hakuneko`;
```

Comment:
This makes it very easy to detect requests from HakuNeko and perform individual actions against the user such as IP block/ban, tracking, ...

Reply:
There won't be any of that.

Reply:
How would someone know?
Nobody has control over the website owners. Maybe they will comply now, but what about in 3 month?

Furthermore less source code is better, especially when it is designed to bypass [the mechanism of stripping the user-agent](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/engine/Request.mjs#L440) that was intentionally implemented in HakuNeko to avoid tracking and detection.

Reply:
I am the website owner.

Reply:
That doesn't affect any of the statements above.

Reply:
OK, don't take my word for it. If users complain that they're getting blocked on either site you can remove the UA.